THIRD-PARTY-NOTICES.TXT

```diff
@@ -0,0 +1,82 @@
+ML.NET uses third-party libraries or other resources that may be
+
+distributed under licenses different than the ML.NET software.
+
+
+
+In the event that we accidentally failed to list a required notice, please
+
+bring it to our attention. Post an issue or email us:
+
+
+
+           dotnet@microsoft.com
```

Comment:
Should this be an ML.NET specific email address?

Reply:
By @shauheen, we use it for now.

Reply:
@eerhardt we do not have a ML.NET one really.